rig/machine_control/machine_controller.py

```diff
@@ -917,6 +935,30 @@ def send_signal(self, signal, app_id=Required):
         self._send_scp(0, 0, 0, SCPCommands.signal, arg1, arg2, arg3)
 
     @ContextMixin.use_contextual_arguments
+    def app_start(self, app_id=Required):
+        """Transmit a signal to start the application."""
+        self.send_signal(consts.AppSignal.start, app_id)
```

Comment:
To leave the paused state (see `load_application`)... it's probably not the *most* useful signal to expose like this, but I'm tempted to keep it anyway.